cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_sameRegionSameNames.ump

```diff
@@ -0,0 +1,25 @@
+class X {
```

Comment:
With the current version of Umple, the following enums are generated when this file is compiled into Java:
From my understanding, the "Sm" prefix indicated the region.
```
//X State Machines
  public enum Sm { s0, s1 }
  public enum SmT1 { Null, t1 }
  public enum SmT2 { Null, t2, t3 }
  public enum SmT1 { Null, t1, t4 }
  public enum SmT2 { Null, t2, t5 }
  private Sm sm;
  private SmT1 smT1;
  private SmT2 smT2;
  private SmT1 smT1;
  private SmT2 smT2;
```